### PR TITLE
Test

### PR DESCRIPTION
**Pull request by pcothenet**
### What

** CHANGEME: What are you changing? **

### Why

** CHANGEME: Why are these changes needed? **

### SDK Release Checklist

- [ ] Have you added an integration test for the changes?
- [ ] Have you built the library locally and made queries against it successfully?
- [ ] Did you update the changelog?
- [ ] Did you bump the package version?
- [ ] If endpoints were removed, did you manually remove the corresponding files? (this should be rare)
- [ ] For breaking changes, did you plan for the release of the new SDK versions and deploy the API to production?

### CODE REVIEW

File: patch_api/api/estimates_api.py [hunk 1]
```diff
@@ -1563,13 +1550,8 @@ def create_sea_shipping_estimate_with_http_info(
         all_params.append("average_daily_balance_btc_sats")
         all_params.append("average_daily_balance_eth_gwei")
         all_params.append("timestamp")
-        all_params.append("destination_country_code")
-        all_params.append("destination_locode")
-        all_params.append("destination_postal_code")
```
Comment (pcothenet) [inline patch_api/api/estimates_api.py:1568]: @holtbp is it possible that you have uncommitted changes to client-code-generation?

File: patch_api/api/estimates_api.py [hunk 2]
```diff
@@ -1205,13 +1200,8 @@ def create_rail_shipping_estimate_with_http_info(
...
-        all_params.append("freight_mass_g")
-        all_params.append("origin_country_code")
-        all_params.append("origin_locode")
-        all_params.append("origin_postal_code")
+        all_params.append("origin_airport")
+        all_params.append("destination_airport")
```
Comment (pcothenet) [inline patch_api/api/estimates_api.py:1204]: @holtbp

Comment (holtbp) [reply patch_api/api/estimates_api.py:1204]: Huh, removing all of the destination/origin fields and putting airport fields for this rail shipping estimate method?

File: patch_api/api/orders_api.py [hunk 3]
```diff
@@ -46,8 +46,8 @@ class OrdersApi(object):
         "average_daily_balance_btc_sats",
         "average_daily_balance_eth_gwei",
         "timestamp",
-        "origin_airport",
-        "destination_airport",
+        "origin_aiport",
+        "destination_aiport",
```
Comment (holtbp) [inline patch_api/api/orders_api.py:50]: Wait, is this a misspelling? Where is that coming from?

File: patch_api/models/create_order_request.py [hunk 4]
```diff
@@ -44,7 +44,7 @@ class CreateOrderRequest(object):
         "currency": "str",
         "amount": "int",
         "unit": "str",
-        "issued_to": "V1OrdersIssuedTo",
+        "issued_to": "IssuedTo",
```
Comment (holtbp) [inline patch_api/models/create_order_request.py:47]: This looks better

File: patch_api/api/orders_api.py (hunk 3 above)
Comment (pcothenet) [reply patch_api/api/orders_api.py:50]: That's what I'm trying to figure out 🤷 . I only made changes to issued_to and ran the generator.

**Pull request closed without merging**